### PR TITLE
epetra: Fix for unguarded SuperLU macro

### PR DESCRIPTION
**Pull request by ndellingwood**
Fix for #5327

### CODE REVIEW

File: packages/epetra/src/Epetra_MultiVector.h [hunk 1]
```diff
@@ -985,6 +985,7 @@ class EPETRA_LIB_DLL_EXPORT Epetra_MultiVector: public Epetra_DistObject, public
   //@}
 
   // Expert-only function
+#undef Reduce
```
Comment (mhoemmen) [inline packages/epetra/src/Epetra_MultiVector.h]: Can you `#undef` something that's not defined?  Perhaps we should do this:
```c++
#ifdef Reduce
#undef Reduce
#endif 
```

Comment (ndellingwood) [reply packages/epetra/src/Epetra_MultiVector.h]: yeah that's safer, I just copy/pasted in rush, thanks!

Comment (ndellingwood) [reply packages/epetra/src/Epetra_MultiVector.h]: Will update in a moment

Comment (ndellingwood) [reply packages/epetra/src/Epetra_MultiVector.h]: @mhoemmen updated, thanks!

**Comment by ndellingwood** [comment]
Updated Epetra_IntMultiVector.h and Epetra_MultiVector.h with guards around `#undef Reduce` as well

**Comment by jhux2** [comment]
@ndellingwood Thanks!

**Pull request merged**